po/de.po

```diff
@@ -148,34 +146,31 @@ msgstr ""
 
 #: newsboat.cpp:163
 msgid "It bundles:"
-msgstr ""
+msgstr "Mitgeliefert wird:"
 
 #: newsboat.cpp:164
-#, fuzzy
 msgid ""
 "- JSON for Modern C++ library, licensed under the MIT License: https://"
 "github.com/nlohmann/json"
 msgstr ""
-"Es wird die unter der MIT-Lizenz stehende Bibliothek JSON für Modernes C++ "
-"verwendet: https://github.com/nlohmann/json"
+"- JSON für Modernes C++, unter der MIT-Lizenz stehende Bibliothek: https://"
+"github.com/nlohmann/json"
 
 #: newsboat.cpp:167
-#, fuzzy
 msgid ""
 "- optional-lite library, licensed under the Boost Software License: https://"
 "github.com/martinmoene/optional-lite"
 msgstr ""
-"Es wird die unter der MIT-Lizenz stehende Bibliothek JSON für Modernes C++ "
-"verwendet: https://github.com/nlohmann/json"
+"- optional-lite, unter der MIT-Lizenz stehende Bibliothek: https://"
```

Comment:
This mentions MIT, but the license is actually BSL. Ditto with expected-lite, below.

Reply:
Oh, indeed! I must have gotten confused by the fuzzy translation.